index.html

```diff
@@ -3,134 +3,477 @@
+        <option value="NE">Nebraska</option>
+        <option value="NH">New Hampshire</option>
+        <option value="NJ">New Jersey</option>
+        <option value="NM">New Mexico</option>
+        <option value="NV">Nevada</option>
+        <option value="NY">New York</option>
+        <option value="OH">Ohio</option>
+        <option value="OK">Oklahoma</option>
+        <option value="OR">Oregon</option>
+        <option value="PA">Pennsylvania</option>
+        <option value="RI">Rhode Island</option>
+        <option value="SC">South Carolina</option>
+        <option value="SD">South Dakota</option>
+        <option value="TN">Tennessee</option>
+        <option value="TX">Texas</option>
+        <option value="UT">Utah</option>
+        <option value="VA">Virginia</option>
+        <option value="VT">Vermont</option>
+        <option value="WA">Washington</option>
+        <option value="WI">Wisconsin</option>
+        <option value="WV">West Virginia</option>
+        <option value="WY">Wyoming</option>
+       </select>
     </div>
   </div>
 </div>
-<div class="main section">
-  <div class="container">
-    <div id="chart"></div>
-  </div>
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-lg-6">
+            <div id="chart1"></div>
+        </div>
+        <div class="col-lg-6">
+            <div id="chart2"></div>
+        </div>
+    </div>
+</div>  
 </div>
 <script type="text/javascript" src="https://npmcdn.com/d3@3.5.17/d3.min.js"></script>
 <script type="text/javascript" src="https://npmcdn.com/science@1.9.3/science.v1.min.js"></script>
 <script type="text/javascript" src="https://npmcdn.com/simple-statistics@2.0.0/dist/simple-statistics.min.js"></script>
+<script type="text/javascript" src="https://npmcdn.com/d3-tip@0.6.7/index.js"></script>
 <script>
   var data = [];
   var options = {
-    yearRange: 'all'
+    state: 'all',
+    region: 'all',
   };
+  var margin = { top: 15, right: 15, bottom: 40, left: 80 };
+  var width = 550 - margin.right - margin.left;
+  var height = 350 - margin.top - margin.bottom;
+  var barMax = 8;
+  
+ // FETCH DATA
+  d3.json('data.json', function (error, json) {
+    if (error) { throw error; }    
+    
+    data = json;
+
+    var charts = [
+       new Chart('#chart1', 'PovertyRate', 'Poverty Rate (%)', 'rate','Arrests Per 100,000 People', '.f', ',0s', '.point'),
+       new Chart('#chart2', 'MedianIncome', 'Median Income', 'rate', 'Arrests Per 100,000 People', '.0s', ',0s','.point2')
+     ];
+
 
-  var svg;
+    // EVENT HANDLERS
+    d3.select('#year-range').on('change', function () {
+      options.state = d3.event.target.value;
+      options.region = 'all';
+      charts.forEach(function (chart) { chart.update();});
+    });
+
+    d3.select('#region-select').on('change', function () {
+      options.region = d3.event.target.value;
+      options.state = 'all';
+      charts.forEach(function (chart) { chart.update();});
+    });
 
```

Comment:
Note that when you select a region ('Northeast') and then a state that is not within that region ('California') the region does not reset. It continues displaying 'Northeast', which doesn't make sense really.